src/InfoBoxes/InfoBoxSettings.hpp

```diff
@@ -124,6 +124,9 @@ struct InfoBoxSettings {
 
   } geometry;
 
+  /* scales the font for InfoBox titles and comments between 80% and 150% */
+  unsigned scale_title_font;
```

Comment:
What does the value mean? This should be documented. Why is this an integer? The example values are `0.8` and `1.5` which cannot be represented with an `unsigned`.

Reply:
I'll change the documentation in the code.